arm/Microsoft.DataProtection/backupVaults/backupPolicies/deploy.bicep

```
@@ -0,0 +1,43 @@
+@description('Required. The name of the backup vault.')
+param backupVaultName string
+
+@description('Optional. The name of the backup policy.')
+param name string = 'DefaultPolicy'
+
+@description('Optional. The properties of the backup policy.')
+param backupPolicyProperties object = {}
+
+
+@description('Optional. Enable telemetry via the Customer Usage Attribution ID (GUID).')
+param enableDefaultTelemetry bool = true
+
+resource defaultTelemetry 'Microsoft.Resources/deployments@2021-04-01' = if (enableDefaultTelemetry) {
+  name: 'pid-47ed15a6-730a-4827-bcb4-0fd963ffbd82-${uniqueString(deployment().name)}'
+  properties: {
+    mode: 'Incremental'
+    template: {
+      '$schema': 'https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#'
+      contentVersion: '1.0.0.0'
+      resources: []
+    }
+  }
+}
+
+resource bv 'Microsoft.DataProtection/backupVaults@2022-03-01' existing = {
+  name: backupVaultName
+}
+
+resource backupPolicy 'Microsoft.DataProtection/backupVaults/backupPolicies@2022-03-01' = {
+  name: name
+  parent: bv
+  properties: backupPolicyProperties
```

Why not call it `properties` as per the resource specification? I assume you wanted to be more specific, but in this case `properties` for a resource `backupPolicies` should do hopefully be explicit enough.

```suggestion
  properties: properties
```

---

Also, in parameter file you also used `properties`